openpype/hosts/maya/plugins/publish/validate_rendersettings.py

```diff
@@ -125,6 +125,10 @@ def get_invalid(cls, instance):
         layer = instance.data['setMembers']
         cameras = instance.data.get("cameras", [])
 
+        if renderer == "_3delight":
+            # 3delight is different enough to need it do it its own validation
+            return invalid
```

Comment:
This validator is already massive, ha! But it stills feels weird we're completely skipping this without any debug log whatsoever.